lib/xeroizer/models/branding_theme.rb

```diff
@@ -5,58 +5,44 @@ module Record
 
     class BrandingThemeModel < BaseModel
 
-      set_permissions :read, :write
-
-      public
-
-      def payment_services(id)
-        @payment_services ||= @application.http_get(@application.client, payment_services_endpoint(id))
-      end
-
-      def add_payment_service(id:, payment_service_id:)
-        xml = {
-          PaymentService: {
-            PaymentServiceID: payment_service_id
-          }
-        }.to_xml
-
-        @application.http_post(@application.client, payment_services_endpoint(id), xml)
+      module Extensions
```

Comment:
Given the namespace change, can you update the CHANGELOG as well / write some quick migration notes?